node/pkg/logscribe/app.go

```diff
@@ -87,3 +131,106 @@ func bulkCopyLogs(ctx context.Context, logsChannel <-chan *[]api.LogInsertModel)
 		}
 	}
 }
+
+func (a *App) processLogs(ctx context.Context) {
+	ticker := time.NewTicker(a.processLogsInterval)
+
+	for {
+		select {
+		case <-ctx.Done():
+			ticker.Stop()
+			return
+		case <-ticker.C:
+			processedLogs := make([]LogInsertModelWithID, 0)
+			logMap := make(map[string]bool, 0)
+			totalLogs := 0
+			for {
+				logs, err := fetchLogs(ctx)
+				if err != nil || len(logs) == 0 {
+					break
+				}
+				totalLogs += len(logs)
+
+				for _, log := range logs {
+					hash := hashLog(log)
+					if !logMap[hash] {
+						processedLogs = append(processedLogs, log)
+						logMap[hash] = true
+					}
+				}
+
+				err = deleteLogs(ctx)
+				if err != nil {
+					break
+				}
+			}
+
+			log.Debug().Msgf("Processed %d logs with %d unique logs", totalLogs, len(processedLogs))
+			if len(processedLogs) > 0 {
+				a.createGithubIssue(ctx, processedLogs)
+			}
+		}
+	}
+}
```

Comment:
instead of running this in certain interval, would it be possible to run this in certain schedule such as every day 11:30 AM or every Friday 11:00 AM. would be more similar to cron jobs